### ISSUE TITLE
Cabnnot select image description

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#303]
## Expected Behavior

Let me select the description added to a file/image and click on it, in case there is a link.

## Current Behavior

Cannot select or click on the description added to a file/image.

## Possible Solution

BTW: Another difference: In the app, the links there are not highlighted and cannot be clicked, in the web client, they are at least highlighted. (but cannot be clicked either)

## Your Environment

* Threema Web version: 1.4.0-gh

**Comment by ovalseven8** [comment]
Yes, it's bad that you cannot copy the image/file description. It's only possible in the image view with dimmed background, but in this view the description is cut after a defined number of chars, so that's not a solution.
